AutomatedTesting/Gem/PythonCoverage/Code/Source/PythonCoverageEditorSystemComponent.cpp

```diff
@@ -163,7 +156,8 @@ namespace PythonCoverage
                 {
                     for (const auto* moduleComponentDescriptor : moduleData.GetModule()->GetComponentDescriptors())
                     {
-                        m_moduleComponents[moduleComponentDescriptor->GetUuid()] = moduleData.GetDebugName();
+                        m_moduleComponents[moduleComponentDescriptor->GetUuid()] =
+                            m_moduleComponents[moduleComponentDescriptor->GetUuid()] = moduleData.GetDebugName();
```

Comment:
Is this a weird PR display issue or something that is currently beyond my understanding? :)

Reply:
Very strange! Possible a strange merge or something. Fixed 👍